app/src/main/java/org/fossasia/openevent/activities/SpeakersActivity.java

```diff
@@ -55,6 +55,8 @@
 
     private boolean flagSocial = false;
 
+    private TextView noSessions;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'noSessions' could be replaced by a local variable.](https://www.codacy.com/app/triplez/open-event-android/file/2406594503/issues/source?bid=3233850&fileBranchId=3233896#l58)